morf-core/src/main/java/org/alfasoftware/morf/jdbc/SqlDialect.java

```diff
@@ -4456,6 +4457,19 @@ protected String getSqlForPortableFunction(Pair<String, List<AliasedField>> func
   /**
    * Returns any statements needed to automatically heal the given schema.
    *
+   * This healer is intended for automated database modifications transparent to the {@link Schema}.

```

Comment:
I think this statement contradicts the rest of the javadoc?  Specifically 'modifications transparent to the Schema', my understanding based on reading the rest of the javadoc is that the method is intended for modifications to things not visible by the Schema?

Reply:
"modifications to things not visible by the Schema" == "modifications transparent to the Schema"
Is that not the same?

Reply:
Ohhh ok I see where you're coming from now, transparent as in see through i.e. invisible?  I was reading it more as transparent as in 'it's clear for it to see'.  Either way, it's perhaps more confusing than saying 'modifications not visible via the Schema interface'.  Or perhaps I am just being dumb... I did work out what the method could be sued from the rest of the javadoc anyway, so I suppose it doesn't matter too much